### PR TITLE
feat(protwailwind): improve articles seo and performance

### PR DESCRIPTION
**Pull request by vojtaholik**
precious articles

### CODE REVIEW

File: apps/protailwind/src/lib/articles.ts [hunk 1]
```diff
@@ -1,5 +1,29 @@
 import {sanityClient} from '../utils/sanity-client'
 import groq from 'groq'
+import z from 'zod'
+
+export const ArticleSchema = z.object({
```
Comment (vojtaholik) [inline apps/protailwind/src/lib/articles.ts:5]: articles are now fully typed

File: apps/protailwind/src/pages/[article].tsx [hunk 2]
```diff
@@ -1,54 +1,51 @@
...
     return {
       notFound: true,
     }
   }
-
-  const article = await getArticle(currentArticle?.slug)
-
   return {
     props: {article},
+    revalidate: 10,
```
Comment (vojtaholik) [inline apps/protailwind/src/pages/[article].tsx:22]: replacing SSR with ISR

File: apps/protailwind/src/templates/article-template.tsx [hunk 3]
```diff
@@ -191,3 +197,20 @@ const Signature = () => {
...
   )
 }
+
+const ArticleMeta: React.FC<{article: Article; shortDescription: string}> = ({
+  article,
+  shortDescription,
+}) => {
+  const {title, date} = article
+  return (
+    <ArticleJsonLd
```
Comment (vojtaholik) [inline apps/protailwind/src/templates/article-template.tsx:207]: adding ArticleJSONLd tags

**Review by DrShpongle — COMMENTED** [review]
Looks good to me!

**Pull request merged**